py/util.go

```diff
@@ -204,3 +205,35 @@ func loadValue(src Object, data interface{}) error {
 	}
 	return nil
 }
+
+// print to gpythons stdout neatly
+func Println(self Object, args ...string) (int, error) {
```

Comment:
perhaps, instead of `self Object`, directly require `self` to be `*Module`, or the correct `Context`?
or, to follow Go's lead, require a "file-like" object? (but admittedly, that function should be called `Fprintln`...)